%YAML:1.0

stereo0:
  cam0:
    cam_overlaps: [1]
    camera_model: pinhole
    distortion_coeffs: [-0.287963527891246, 0.06114542861916172, -0.0013978746355641754, -0.002181166008501264]
    distortion_model: radtan
    intrinsics: [565.3721920866982, 505.1447045230691, 683.6162155364156, 349.48958654750106]
    resolution: [1280, 640]
    rostopic: /cam0/image_raw
  cam1:
    T_cn_cnm1:
      - [0.9997320642668582, -5.2825441131717726e-05, 0.023147286799775978, -0.06902088678479362]
      - [-4.6984623034969e-05, 0.9999907048105098, 0.004311390149006496, 0.0004583467012251034]
      - [-0.023147299392445593, -0.004311322540070649, 0.9997227690859021, -0.0018165169427550925]
      - [0.0, 0.0, 0.0, 1.0]
    cam_overlaps: [0]
    camera_model: pinhole
    distortion_coeffs: [-0.2870363321003373, 0.060581455066596394, -0.0009602203554854264, -0.00029592095252132145]
    distortion_model: radtan
    intrinsics: [565.9162117034637, 504.9439955809892, 643.7698814877796, 365.7936253994491]
    resolution: [1280, 640]
    rostopic: /cam1/image_raw